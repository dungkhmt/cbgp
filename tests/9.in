50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
