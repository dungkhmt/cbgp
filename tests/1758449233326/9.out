(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minAngle=0.10947225997045784 
sumAngle=46.6992660781195 
minEdgeLength=1.4142135623730951 
sumEdgeLength=3177.9874813687834 
minDistance=0.04471679684432152 
sumDistance=293435.3139784338 

0.0000000000 0.1094722600 46.6992660781 1.4142135624 3177.9874813688 0.0447167968 293435.3139784338
0: (102, 34),
1: (20, 17),
2: (68, 93),
3: (77, 108),
4: (101, 72),
5: (109, 1),
6: (79, 55),
7: (69, 112),
8: (0, 0),
9: (61, 112),
10: (200, 77),
11: (91, 123),
12: (25, 11),
13: (60, 99),
14: (87, 125),
15: (34, 46),
16: (70, 72),
17: (107, 3),
18: (99, 16),
19: (87, 44),
20: (28, 27),
21: (89, 38),
22: (67, 73),
23: (96, 17),
24: (94, 108),
25: (85, 38),
26: (70, 109),
27: (44, 49),
28: (140, 8),
29: (88, 107),
30: (87, 39),
31: (36, 22),
32: (107, 92),
33: (8, 1),
34: (185, 18),
35: (90, 130),
36: (63, 110),
37: (92, 25),
38: (41, 50),
39: (102, 103),
40: (68, 92),
41: (96, 111),
42: (60, 62),
43: (114, 102),
44: (17, 5),
45: (59, 145),
46: (95, 114),
47: (91, 129),
48: (103, 104),
49: (98, 93),
102 34 20 17 68 93 77 108 101 72 109 1 79 55 69 112 0 0 61 112 200 77 91 123 25 11 60 99 87 125 34 46 70 72 107 3 99 16 87 44 28 27 89 38 67 73 96 17 94 108 85 38 70 109 44 49 140 8 88 107 87 39 36 22 107 92 8 1 185 18 90 130 63 110 92 25 41 50 102 103 68 92 96 111 60 62 114 102 17 5 59 145 95 114 91 129 103 104 98 93 
0: (102, 34), 1: (20, 17), 2: (68, 93), 3: (77, 108), 4: (101, 72), 5: (109, 1), 6: (79, 55), 7: (69, 112), 8: (0, 0), 9: (61, 112), 10: (200, 77), 11: (91, 123), 12: (25, 11), 13: (60, 99), 14: (87, 125), 15: (34, 46), 16: (70, 72), 17: (107, 3), 18: (99, 16), 19: (87, 44), 20: (28, 27), 21: (89, 38), 22: (67, 73), 23: (96, 17), 24: (94, 108), 25: (85, 38), 26: (70, 109), 27: (44, 49), 28: (140, 8), 29: (88, 107), 30: (87, 39), 31: (36, 22), 32: (107, 92), 33: (8, 1), 34: (185, 18), 35: (90, 130), 36: (63, 110), 37: (92, 25), 38: (41, 50), 39: (102, 103), 40: (68, 92), 41: (96, 111), 42: (60, 62), 43: (114, 102), 44: (17, 5), 45: (59, 145), 46: (95, 114), 47: (91, 129), 48: (103, 104), 49: (98, 93), 
numberIntersection=0 minAngle=0.10947225997045784 sumAngle=46.69926607805175 minEdgeLength=1.4142135623730951 sumEdgeLength=3177.9874813687716 minDistance=0.04471679684432152 sumDistance=293435.31397843873 
Time taken: 33.395 s
